notifications.bs

```diff
@@ -483,7 +516,8 @@ must be run.
 <ol>
   <li><p>Wait for any <a lt="fetch">fetches</a> to complete and
   <var>notification</var>'s <a>icon resource</a> and <a>sound resource</a>
-  to be set (if any).
+  to be set (if any), as well as the <a>action icon resource</a>s for the
```

Comment:
As it turns out, Bikeshed seems to be clever about plurals, it just links them correctly.